labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +99,6 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    lines.map(_.split(" ").map(_.length).max).max
```

Comment:
Although not necessary (and may in fact it might hurt readability), it might be useful to experiment with removing some () and . that Scala can situationally omit.